### PR TITLE
Made BasketService Functional, Added new functionality

### PR DESCRIPTION
**Pull request by DanielRailean**
1. Changed Basket service Interface
    *Made Service accept Recipe instead of RecipeItem
    *Added method to get added recipes
2. Changed Implementation of basket service.
3. Changed Index page to make use of the updated service
4. Added option to Clear cart to navbar
5. Updated startup to add neccesary configuration for localStorage

### CODE REVIEW

File: Presentation Tier/WebApp/Startup.cs [hunk 1]
```diff
@@ -35,6 +35,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IIngredientService, IngredientService>();
             services.AddScoped<IBasketService, BasketService>();
             services.AddBlazoredLocalStorage();
```
Comment (markvinczehu) [inline Presentation Tier/WebApp/Startup.cs:37]: maybe then this should be deleted

**Review by markvinczehu — COMMENTED** [review]
I commented on one of the lines that I think is a duplication

File: Presentation Tier/WebApp/Startup.cs (hunk 1 above)
Comment (markvinczehu) [reply Presentation Tier/WebApp/Startup.cs:37]: line 37

**Review by DanielRailean — COMMENTED** [review]
Both lines are actually needed .

**Review by markvinczehu — APPROVED** [review]
Good solution with the cookies

**Review by DanielRailean — COMMENTED** [review]
You approved changes to read me but haven't read them as I see.

**Pull request closed without merging**